sacred/config/custom_containers.py

```diff
@@ -165,8 +165,7 @@ def revelation(self):
 
 
 class ReadOnlyContainer:
-    def __init__(self, *args, message=None, **kwargs):
-        super().__init__(*args, **kwargs)
+    def __init__(self, message=None):
         self.message = message or "This container is read-only!"
```

Comment:
We might want to hide the message attribute better by calling it `_message` or even `__message`. That's something that should be private and not used or modified by users of sacred.

Reply:
Sounds good